README.md

```diff
@@ -58,6 +58,17 @@ We also encourage to use [husky](https://github.com/typicode/husky) and [lint-st
 }
 ```
 
+### Ignoring Files
+
+You can ignore files by creating a `.eslintignore` in your directory, and using standard glob patterns. `prettier-standard` will not run on these files.
+
+#### .eslintignore
+```
+myScript.js
+myFolder/**/*
+...etc
+```
```

Comment:
Maybe skip this if we follow the above?

Reply:
Personally, I think having the example right there is helpful - I think the ESlint doc throws a lot at you when you first hit the page. I added another example to show the package.json method as well, but we can remove both examples if you think it's extraneous